scripts/deploy_core.ts

```diff
@@ -179,4 +188,27 @@ async function uploadAndInitMaker(terra: LCDClient, wallet: any) {
     }
 }
 
+async function uploadAndInitWhiteList(terra: LCDClient, wallet: any) {
+    const network = readArtifact(terra.config.chainID)
+
+    if (!network.whiteListAddress) {
+        console.log('Deploying white list...')
+        let resp = await deployContract(
+            terra,
+            wallet,
+            network.multisigAddress,
+            join(ARTIFACTS_PATH, 'astroport_whitelist.wasm'),
+            {
+                admins: ["terra17lmam6zguazs5q5u6z5mmx76uj63gldnse2pdp", "terra1757tkx08n0cqrw7p86ny9lnxsqeth0wgp0em95"],
```

Comment:
What addresses are set here?
It seems like contracts will instantiate whitelists themselves